thunder/torch/experimental/dtensor_utils.py

```diff
@@ -58,14 +58,17 @@ def materialize_fake_tensors(t):
                 return t
 
             if isinstance(t, DTensorProxy):
-                i_t = torch.randn(
+                i_t = torch.ones(
```

Comment:
`torch.randn` only works for floating and complex dtypes but not integer. `torch.ones` works for all.